Platforms/QemuQ35Pkg/Tcg/Tcg2ProtocolStub/Tcg2ProtocolStub.inf

```diff
@@ -0,0 +1,36 @@
+## @file
+# This module installs the gEfiTcg2ProtocolGuid to satisfy
+# DEPEX expressions in modules which do not actually use the
+# protocol.
+#
+# Copyright (C) 2021, Advanced Micro Devices, Inc.

```

Comment:
Is this a 2021 AMD module?

Reply:
The .c file has a Microsoft only copyright so it seems strange.

Reply:
I copied and modified a random module and changed the copyright in the .c but not in the inf. Will update.